sapi/fpm/fpm/fpm_conf.c

```diff
@@ -915,8 +915,7 @@ static int fpm_conf_process_all_pools(void)
 			}
 
 			if (config->listen_backlog < FPM_BACKLOG_DEFAULT) {
-				zlog(ZLOG_WARNING, "[pool %s] listen.backlog(%d) was too low for the ondemand process manager. I updated it for you to %d.", wp->config->name, config->listen_backlog, FPM_BACKLOG_DEFAULT);
-				config->listen_backlog = FPM_BACKLOG_DEFAULT;
+				zlog(ZLOG_WARNING, "[pool %s] listen.backlog(%d) is too low for the ondemand process manager. I suggest updating it to at least %d.", wp->config-  >name, config->listen_backlog, FPM_BACKLOG_DEFAULT);
```

Comment:
The build failure is legit, comes from the `wp->config-` `>name` typo.

Reply:
Thanks! Maybe 3rd time is the charm. :)